### ISSUE TITLE
Add support to select what server to use

### ISSUE THREAD

**Issue by sunkan** [anttiviljami/openapi-client-axios#28]
At the moment the client is hard coded to use the first server in the definition regardless of how many are defined.

Sure it can be override by the `axiosConfigDefaults.baseURL` but then you have to manually find out the url or extract it from the document it would be much nicer to something like this
```
const api = new OpenAPIClientAxios({
  definition: 'https://example.com/api/openapi.json'
  server: 1
});

//or by method
api.selectServer(1);
//or by description
api.selectServer('Eu Server');

```

**Comment by anttiviljami** [comment]
I love it! Here's my suggestion for the interface

```javascript
const api = new OpenAPIClientAxios({
  definition: 'https://example.com/api/openapi.json',
  withServer: 1,
});

//or by method
api.withServer(1);
//or by description
api.withServer('Eu Server');
//or by full server object (override)
api.withServer({ url: 'http://eu.api.com', description: 'Eu Server' });
```

**Comment by anttiviljami** [comment]
This feature was released as part of `openapi-client-axios@3.5.0`

https://github.com/anttiviljami/openapi-client-axios/blob/master/DOCS.md#withserverserver
